### ISSUE TITLE
Multilangual citations

### ISSUE THREAD

**Issue by foss-friendly** [zotero/zotero#1529]
Dear Zotero Community, 

I'm a researcher in Humanities and in my field of studies "Multilingual citations" are a must, in order for the academic text to be considered "valid". I know that this is a old Feature Request for Zotero but unfortunately it still isn't implemented, thus preventing wide-spread usage of Zotero in my Faculty. The idea is the following: if we have one English, one French book and one p.ex. Serbian / Croatian book, the footnote shall look like "New footnote" **In:** Multilingual Zotero, **p.** 120; "Nouveaux amis" **A:** "La France de demain", p. 120; "Danas" **U:** "Novi razpored", **str.** 120. Can Zotero be adapted for such a usage case: before creating a footnote, check for the Language property of the book entry in the Collection database, and according to the Language property apply the corresponding localized "footnote parts"?  

Thank You for all Your time spent on this wondefull _lifesaving_ program!

**Comment by dstillman** [comment]
If you need multilingual citations, you can use [Juris-M](https://juris-m.github.io/), a third-party fork of Zotero. This is unlikely to be implemented in Zotero itself.

For further support queries, please post to the [Zotero Forums](https://forums.zotero.org).

**Comment by foss-friendly** [comment]
I never managed to get Juris-M to work in both MS Office on Windows and LibreOffice on Linux in the same document (230 pages and 1000+ footnotes). I tried a simple copy-paste of the "multilangual citations" parts of Juris code on top of Zotero's, and couldn't build it.... Anyway, thank You for Your reply. Unfortunately the word "unlikely" is very disturbing and I might be forced to look for another bibliography manager solution, although I love Zotero and contribute translations to it from time to time.

**Comment by foss-friendly** [comment]
I never managed to get Juris-M to work in both MS Office on Windows and LibreOffice on Linux in the same document (230 pages and 1000+ footnotes). I tried a simple copy-paste of the "multilangual citations" parts of Juris code on top of Zotero's, and couldn't build it.... Anyway, Thank You for Your reply. Unfortunately the word "unlikely" is very disturbing... I also searched for other solutions, but it looks like all the good ones use CSL, which decided to abandon all user requests for this must-have feature. I thought at first that this is an issue of Zotero, but clearly I was wrong. I'll continue to contribute translations if my new transifex acc gets approved, and I'll hope that CSL devs will at some point find a solution to this problem.
